tests/zip.spec.ts

```diff
@@ -78,7 +78,7 @@ describe('zip support', function() {
 	);
 
 	describe('compression method', function() {
-		const compressionMethods = [
+		const compressionMethods: string[] = [
```

Comment:
Is this change actually necessary? I don't mind it but generally I leave typescript to do the inference as it makes changes a bit easier

Reply:
debugging leftover, removed